CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 ### Internal
```

Comment:
There's a typo in the commit message, there's a missing 'N' in the 'CHANGELOG' word.